includes/tutor-general-functions.php

```diff
@@ -1023,5 +1023,29 @@ function( $classes ) use ( $custom_class ) {
 			);
 		}
 	}
+
+}
+
+if ( ! function_exists( 'tutor_close_able_alert_msg' ) ) {
+	/**
+	 * Create a close-able alert message
+	 *
+	 * @since 2.1.9
+	 *
+	 * @param string $message alert message.
+	 * @param string $alert alert key like: success, warning, danger, etc.
+	 *
+	 * @return void
+	 */
+	function tutor_close_able_alert_msg( string $message, string $alert = 'success' ) {
+		?>
+		<div class="tutor-alert tutor-<?php echo esc_attr( $alert ); ?> tutor-mb-12 tutor-alert tutor-success tutor-mb-12 tutor-d-flex tutor-align-center tutor-justify-between">
+			<span>
+				<?php echo esc_html( $message ); ?>
```

Comment:
We can allow basic HTML tags so that, we can show alert messages in various text formats.